### DISCUSSION TITLE
"Unknown resource provided"

### DISCUSSION THREAD

**Discussion by comblock** [HypixelDev/PublicAPI#437] [Q&A] [answered]
When I request the current player count with the URL:  "https://api.hypixel.net/resources/counts?key=" + API_KEY
It returns {'success': False, 'cause': 'Unknown resource provided'}
What does this error mean and how can I fix it?

**Comment by ConnorLinfoot** [comment, marked as answer]
The endpoint for player counts is just `/counts`, `/resources` is for semi-static resources such as achievements. https://api.hypixel.net/#tag/Other/paths/~1counts/get
